vector/src/main/res/layout/item_room_placeholder.xml

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
there is a problem of colors in dark mode. Same for all placeholders I guess

Reply:
it's using vctr_list_header_background_color, not sure which one to take